tests/flow/test_point.py

```diff
@@ -143,3 +143,45 @@ def test_point_index_lookup(self):
         self.env.assertEquals(len(idx_res), 1)
         self.env.assertEquals(idx_res[0][0], 'univ')
 
+    def test_nested_point(self):
+        expected_value = [{'latitude':32, 'longitude':34}]
+        # point as an array element
+        q = "RETURN [point({latitude:32, longitude:34})]" 
+        res = redis_graph.query(q)
+        self.env.assertEquals(res.result_set[0][0], expected_value)
+
+        # test setting an array with a point as an entity attribute
+        q = "CREATE (n:L {v: [point({latitude:32, longitude:34})]}) RETURN n.v"
+        res = redis_graph.query(q)
+        self.env.assertEquals(res.result_set[0][0], expected_value)
+
+        # update expected value
+        expected_value = [{'latitude':34, 'longitude':35}]
+        # update existing node, place a different point in its array
+        q = "MATCH (n:L) SET n.v = [point({latitude:34, longitude:35})] RETURN n.v"
+        res = redis_graph.query(q)
+        self.env.assertEquals(res.result_set[0][0], expected_value)
+
+        # order a list of points
+        expected_value = [
+        {'latitude':32, 'longitude':31},
+        {'latitude':32, 'longitude':32},
+        {'latitude':33, 'longitude':35},
+        {'latitude':29, 'longitude':36}
```

Comment:
Use two points with the same longitude value to test latitude comparison (will suppress Codecov complaint).